pkg/pillar/kubeapi/nitoapiserver.go

```diff
@@ -1,12 +1,10 @@
-// Copyright (c) 2024 Zededa, Inc.
```

Comment:
Self review. I have no idea on how copyright info is deleted. I will fix that

Reply:
I see it is still missing.

Reply:
Ack, it will be in next commit.